Sources/UILabelExtensions.swift

```diff
@@ -11,7 +11,20 @@
 import UIKit
 
 extension UILabel {
+
+    /// EZSwiftExtensions: fontSize: 17
+    public convenience init(x: CGFloat, y: CGFloat, w: CGFloat, h: CGFloat) {
+        self.init(x: x, y: y, w: w, h: h, fontSize: 17)
```

Comment:
I am pretty certain I removed this a few commits ago. Lets make the other convenience init default fontSize = 17.